python_src/Dockerfile

```
@@ -13,11 +13,18 @@ RUN apt-get install -y libpq-dev gcc curl
 RUN pip install -U pip
 RUN pip install "poetry==1.1.14"
 
-# Copy src dir and pyproject file and create a virtualenv with all needed dependenciies using poetry
-COPY . /lamp/
+# Create a directory for project to live in
+RUN mkdir lamp/
+
+# copy poetry and pyproject files and install dependencies
 WORKDIR /lamp/
+COPY poetry.lock poetry.lock
+COPY pyproject.toml pyproject.toml
 RUN poetry install --no-dev --no-interaction --no-ansi -v
 
+# Copy src directory to run against
+COPY src src
```

Just need to run `poetry install` again to install the `lamp_py` module

```suggestion
COPY src src
RUN poetry install --no-dev --no-interaction --no-ansi -v
```